### PR TITLE
Add beautifulsoup4 (w/ soupsieve)

### PR DESCRIPTION
**Pull request by msabramo**
Example:

If you do:

```js
languagePluginLoader.then(() => {
  // pyodide is now ready to use...
  pyodide.loadPackage('beautifulsoup4').then(() => {
    pyodide.runPython(`
text = """
<div>
  <!-- These are animals -->
  <p class="a">Cat</p>
  <p class="b">Dog</p>
  <p class="c">Mouse</p>
</div>
"""

from bs4 import BeautifulSoup
import soupsieve
soup = BeautifulSoup(text, 'html.parser')

print(soup.select('p:is(.a, .c)'))
    `);
  });
});
```

then the output is:

```
[<p class="a">Cat</p>, <p class="c">Mouse</p>]
```

### CODE REVIEW

File: packages/beautifulsoup4/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version: 4.7.1
+source:
+  sha256: 945065979fb8529dd2f37dbb58f00b661bdbcbebf954f93b32fdf5263ef35348
+  url: https://files.pythonhosted.org/packages/80/f2/f6aca7f1b209bb9a7ef069d68813b091c8c3620642b568dac4eb0e507748/beautifulsoup4-4.7.1.tar.gz
+requirements:
+  run:
+  - soupsieve
+test:
+  imports:
+  - beautifulsoup4
```
Comment (mdboom) [inline packages/beautifulsoup4/meta.yaml]: I think:

```suggestion
  - BeautifulSoup4
```

Comment (msabramo) [reply packages/beautifulsoup4/meta.yaml]: Updated. Thanks!

🤞

Comment (msabramo) [reply packages/beautifulsoup4/meta.yaml]: Hmmm, I wonder if this should be `bs4` actually?

Comment (mdboom) [reply packages/beautifulsoup4/meta.yaml]: Good point.  I think so.

**Comment by msabramo** [comment]
`test-chrome` error looks like some kind of intermittent network error?

**Pull request merged**